metapool/sample_sheet.py

```diff
@@ -388,6 +353,145 @@ def merge(self, sheets):
                 else:
                     pass
 
+    def _remap_table(self, table, strict=True):
+        '''
```

Comment:
I cannot find where the `assay` variable is populated for this method.  The pre-refactoring version had `assay` as a parameter but this one does not ... where is it specified?

Reply:
When the function was global, we needed to pass the assay type in order to return the proper dict of key/value pairs for remapping. Now that each class has its own self.remapper, the assay type doesn't need to be passed as each child class defines the self.remapper member that will be used by the method shared between them.